src/main/java/core/basesyntax/service/FileWriter.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.io.File;
+
+public interface FileWriter {
+    void write(String resultingReport, File file);
```

Comment:
Consider accepting the file path as a String rather than a File object. This would allow more flexibility in how the file location is determined and used within the implementing classes.